riak/client/index_page.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 from collections import namedtuple
-from collections.abc import Sequence
```

Comment:
Looks like this change is wrong:

```
$ python3.9
Python 3.9.10 (main, Jan 15 2022, 11:48:04) 
[Clang 13.0.0 (clang-1300.0.29.3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from collections import Sequence
<stdin>:1: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.10 it will stop working
```

Reply:
fixed

Reply:
all other imports do not generate deprecation warning

Reply:
Hmm, they do for me:

```$ python3.9
Python 3.9.10 (main, Jan 15 2022, 11:48:04) 
[Clang 13.0.0 (clang-1300.0.29.3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from collections import Mapping
<stdin>:1: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.10 it will stop working```

Reply:
oh, the warning shows only for the very first import only. got it

Reply:
fixed